$name__norm$-api/src/main/scala/$package$/api/$name__Camel$Service.scala

```diff
@@ -23,21 +29,41 @@ trait $name;format="Camel"$Service extends Service {
     */
   def useGreeting(id: String): ServiceCall[GreetingMessage, Done]
 
+
+  /**
+    * This gets published to Kafka.
+    */
+  def greetingsTopic(): Topic[GreetingMessage]
+
   override final def descriptor = {
     import Service._
     // @formatter:off
-    named("$name;format="norm"$").withCalls(
-      pathCall("/api/hello/:id", hello _),
-      pathCall("/api/hello/:id", useGreeting _)
-    ).withAutoAcl(true)
+    named("$name;format="norm"$")
+      .withCalls(
+        pathCall("/api/hello/:id", hello _),
+        pathCall("/api/hello/:id", useGreeting _)
+      )
+      .withTopics(
```

Comment:
`publishing` should be deprecated in favour of `withTopics`. https://github.com/lagom/lagom/issues/769